samples/zigbee/light_bulb/README.rst

```diff
@@ -0,0 +1,108 @@
+.. _zigbee_light_bulb_sample:
+
+Zigbee: Light bulb
+##################
+
+This Zigbee light bulb sample demonstrates a simple light bulb whose brightness can be regulated by a different device.
+
+You can use this sample together with :ref:`Zigbee network coordinator <zigbee_network_coordinator_sample>` and Zigbee light switch to set up a basic Zigbee network.
+
+Overview
+********
+
+The Zigbee light bulb sample assumes the Zigbee Router role and implements the Dimmable Light profile.
+This profile allows to change the brightness level of a LED on the light bulb development kit.
+
+In the default sample configuration, the changes to the light bulb brightness are reflected on LED 4.
+
+Requirements
+************
+
+* One or more of the following development kits:
+
+  * |nRF52840DK|
+  * |nRF52840Dongle|
+  * |nRF52833DK|
+
+* The :ref:`Zigbee network coordinator <zigbee_network_coordinator_sample>` application programmed on one separate device.
+* The Zigbee light switch application programmed on one or more separate devices.
+
+You can mix different development kits.
+
+User interface
+**************
+
+LED 1 and LED 2:
+    Indicates the |BLE| status:
+
+    * Blinking - |BLE| advertising ongoing.
+    * Slow blinking (period of 200 ms when on, period of 800 ms when off) - Device is not connecting to a Zigbee mesh network.
+    * Rapid blinking (period of 100 ms) - Device is connecting to a Zigbee mesh network.
+    * Solid - Device is connected over |BLE|.
+
+LED 3:
+    Indicates whether the network is open or closed.
+
+    .. note::
+         When you use the light bulb on the |nRF52840Dongle|, it is the LED 3 that informs about the successful network joining.
```

Comment:
Not sure I understand ... "successful network joining" is the same as line 41? But on the dongle it's indicated by LED 3 instead of 1&2? And is there then no indication of network open or closed?

Reply:
This note triggered more extensive edits. Fixed now.